custom/icds_reports/utils/aggregation_helpers/tests/sql_output/agg-awc-daily.distributed.txt

```diff
@@ -11,7 +11,7 @@ DROP TABLE IF EXISTS "agg_awc_daily_2019-01-01"
         INSERT INTO "agg_awc_daily_2019-01-01" (
             state_id, district_id, block_id, supervisor_id, awc_id, aggregation_level, date, cases_household, cases_person, cases_person_all, cases_person_has_aadhaar, cases_person_beneficiary, cases_child_health, cases_child_health_all, cases_ccs_pregnant, cases_ccs_pregnant_all, cases_ccs_lactating, cases_ccs_lactating_all, cases_person_adolescent_girls_11_14, cases_person_adolescent_girls_15_18, cases_person_adolescent_girls_11_14_all, cases_person_adolescent_girls_15_18_all, daily_attendance_open, num_awcs, num_launched_states, num_launched_districts, num_launched_blocks, num_launched_supervisors, num_launched_awcs, cases_person_has_aadhaar_v2, cases_person_beneficiary_v2, state_is_test, district_is_test, block_is_test, supervisor_is_test, awc_is_test
         ) (SELECT
-            state_id, district_id, block_id, supervisor_id, awc_id, aggregation_level, %(date)s, cases_household, cases_person, cases_person_all, cases_person_has_aadhaar, cases_person_beneficiary, cases_child_health, cases_child_health_all, cases_ccs_pregnant, cases_ccs_pregnant_all, cases_ccs_lactating, cases_ccs_lactating_all, cases_person_adolescent_girls_11_14, cases_person_adolescent_girls_15_18, cases_person_adolescent_girls_11_14_all, cases_person_adolescent_girls_15_18_all, 0, num_awcs, 0, 0, 0, 0, 0, cases_person_has_aadhaar_v2, cases_person_beneficiary_v2, state_is_test, district_is_test, block_is_test, supervisor_is_test, awc_is_test
```

Comment:
These zeroes are changed to the actual column because I passed 'use_agg_awc': True below as test argument.